akka-persistence/src/test/scala/akka/persistence/PersistentActorStashingSpec.scala

```diff
@@ -108,9 +114,13 @@ abstract class PersistentActorStashingSpec(config: Config) extends PersistenceSp
       persistentActor ! Cmd("a")
       persistentActor ! Cmd("b")
       persistentActor ! Cmd("c")
+      persistentActor ! Cmd("d")
+      persistentActor ! Cmd("e")
```

Comment:
A separate test covering this bug would be nicer than appending it to existing test cases.